process-engine/process_engine_runtime
# ✨ 🚧  [WIP]: Add limit and offset to list queries

## Changes

1. Add `offset` and `limit` as optional parameters to each query that returns a list.

## Issues

Closes #379

PR: #398

## How to test the changes

- Create a large number of entries in any of your databases (i.e. start numerous ProcessInstances in varying correlations)
- Make a request for getting a list of anything and provide `limit` and `offset`
- See that the returned list is reduced, according to the `limit` and `offset` you provided
